tasks/install.yml

```diff
@@ -0,0 +1,101 @@
+---
+- name: Install required apt packages
```

Comment:
Add update cache task that is not changed

Reply:
fixed